studiopress/atomic-blocks
# Add e2e test setup and test of blocks

- [x] Adds an e2e setup, mainly using Gutenberg functions and scripts, like `wp-env` and `wp-scripts`.
- [x] Tests user input in 1 block
- [x] Adds a job to `.circleci/config.yml` for this

### How to test

1. Ensure Docker Desktop is running 
2. npm install && npm run build-assets
3. npm run env start
4. npm run test:e2e # Or npm run test:e2e:interactive to see the test run.
5. npm run env stop

### Documentation
PR includes (basic) documentation.

### Suggested changelog entry
- Add a lightweight e2e setup and a test of blocks.

### Notes
Feel free to comment, especially if there's something that would be good to test.
